content/posts/annoucing-slight-v0_3.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Annoucing Slight v0.3"
+description: "Announcing new features and updates on Slight v0.3 release."
+date: 2023-02-25 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaiao_zhou"
+tags: ["slight", "spiderlightning", "Wasm", "WASI"]
+---
+
+We are excited to announce the release of `Slight` v0.3, the latest version of our WebAssembly runtime that utilizes SpiderLightning (also known as `wasi-cloud`) capabilities.
+
+You can download the latest release by running the following command if you are using Linux or macOS:
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/deislabs/spiderlightning/main/install.sh)"
+```
+
+If you are using Windows, you can download the latest release by running this command:
+
+```powershell
+iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/deislabs/spiderlightning/main/install.ps1'))
+```
+
+This release includes several new features, such as the ability to author HTTP clients as Wasm modules, publish or receive events, query databases with SQL, and much more. You can find all the example Slight applications in [here](https://github.com/deislabs/spiderlightning/tree/main/examples). Now, let’s dive into it!
```

Comment:
Just a general note about making accessible websites. Avoid making the hyperlinked word so generic that if someone using a screen reading is scanning for links to nagivate your site that they just get "here" or "click this", etc. It makes it really hard to navigate with a screenreader.

It really helps people out to make the link text descriptive enough that if you were only looking at a list of hyperlinked words, you could figure out where the link would go to. In this case, a little change makes a big difference in accessibility:
```suggestion
This release includes several new features, such as the ability to author HTTP clients as Wasm modules, publish or receive events, query databases with SQL, and much more. You can find all the [example Slight applications in our repository](https://github.com/deislabs/spiderlightning/tree/main/examples). Now, let’s dive into it!
```